pkg/names/names.go

```diff
@@ -2,3 +2,6 @@ package names
 
 // ManifestDir is the directory where manifests are located.
 const ManifestDir = "./bindata"
+
+// NMstateResourceName is the name of the CR that the operator will reconcile
+const NMstateResourceName = "nmstate"
```

Comment:
IIUC, the user should create the NMstate instance, so I think It would be useful to document this requirement somewhere.

Reply:
Well, yes and no. Under CVO, that would not be the case. CNO does it this way. I will see how they document it.